src/natcap/invest/urban_nature_access.py

```diff
@@ -337,11 +350,179 @@ def execute(args):
             greenspace_population_ratio_task,
         ])
 
+    # SUP_DEMi_cap
+    per_capita_greenspace_budget_task = graph.add_task(
+        pygeoprocessing.raster_calculator,
+        kwargs={
+            'base_raster_path_band_const_list': [
+                (file_registry['greenspace_supply'], 1),
+                (float(args['greenspace_demand']), 'raw')
+            ],
+            'local_op': _greenspace_budget_op,
+            'target_raster_path': file_registry['greenspace_budget'],
+            'datatype_target': gdal.GDT_Float32,
+            'nodata_target': FLOAT32_NODATA
+        },
+        task_name='Calculate per-capita greenspace budget',
+        target_path_list=[file_registry['greenspace_budget']],
+        dependent_task_list=[
+            convolved_greenspace_population_ratio_task
+        ])
+
+    # SUP_DEMi
```

Comment:
Is the ending `i` leftover from a vim insert 😄 ?

Reply:
Normally it would be!  In this case, it's the per-pixel notation they're using in the UG chapter, so I added a bigger comment to make it look a little more deliberate :) a2b4e2c5